packages/edge-config/src/index.ts

```diff
@@ -208,17 +271,25 @@ export const createClient = trace(
      * While in development we use SWR-like behavior for the api client to
      * reduce latency.
      */
-    const shouldUseSwr =
+    const useDevelopmentCache =
```

Comment:
We shouldn't prefix with "use" due to confusion with react hooks. Not sure it would actually lead to errors, but better to avoid this in the first place

```suggestion
    const shouldUseDevelopmentCache =
```